spikeinterface/extractors/nwbextractors.py

```diff
@@ -106,7 +106,7 @@ def __init__(self, file_path: PathType, electrical_series_name: str = None, load
             sampling_frequency = 1. / np.median(np.diff(timestamps[:samples_for_rate_estimation]))
 
         if load_time_vector and timestamps is not None:
-            times_kwargs = dict(time_vector=self._es.timestamps)
+            times_kwargs = dict(sampling_frequency=sampling_frequency, time_vector=self._es.timestamps)
```

Comment:
No! time_vector and sampling_frequencu are mutualy exclusive and use only for SegmentRecording.
We must reverse this.